python/llm/example/GPU/HuggingFace/LLM/codegeex2/README.md

```diff
@@ -27,10 +26,20 @@ conda activate llm
 
 # below command will install intel_extension_for_pytorch==2.1.10+xpu as default
 pip install --pre --upgrade ipex-llm[xpu] --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/us/
-pip install transformers==4.31.0
 ```
 
-### 2. Configures OneAPI environment variables for Linux
+### 2. Replace File
+If you select the codegeex2-6b model ([THUDM/codegeex-6b](https://huggingface.co/THUDM/codegeex2-6b)), please note that their code (`tokenization_chatglm.py`) initialized tokenizer after the call of `__init__` of its parent class, which would result in error. To address issue, we have provided updated file ([tokenization_chatglm.py](./tokenization_chatglm.py))
```

Comment:
If you select the codegeex2-6b model ([THUDM/codegeex-6b](https://huggingface.co/THUDM/codegeex2-6b)), please note that their code (`tokenization_chatglm.py`) initialized tokenizer after the call of `__init__` of its parent class, which may result in error during loading tokenizer. To address this issue, we have provided an updated file ([tokenization_chatglm.py](./tokenization_chatglm.py))